### DISCUSSION TITLE
How do I find my collective's Public ID

### DISCUSSION THREAD

**Discussion by ZoeLocke** [opencollective/opencollective#6756] [Q&A] [answered]
Update: I've confirmed the ID isn't actually required; slug is sufficient. It still would be good to know how to find it, though.

---

Feeling dense asking this; it seems to be such an obvious thing that I can't find any mention of it anywhere online 😅

Where can I find my collective's public ID? According to the API Docs, Account has an attribute of ID and AFAIK it's needed to query an account properly. I'm actually looking to find the account ID of a project within the collective; I assume they're the same?

![image](https://github.com/opencollective/opencollective/assets/4498165/084cc118-c1ef-46f4-8a63-11912304fac0)

**Comment by znarf** [comment, marked as answer]
Your answer is right, in most cases (all), you don't need to know the `id` and can just query with the `slug`. This is the way to go. 

If you need to know the `id` from a `slug`, you first query the account (with the slug) and ask for the id.

```graphql
query {
  account(slug: "backyourstack") {
    id
  }
}
```

**Reply by ZoeLocke** [reply]
Brilliant, thank you ☺
